nss-day-cohort-41/tabloid-fullstack-shocking-secret-keepers
# Sf edit/delete

# Description
This pull request refers to issues in the following tickets:
Ticket #19 
Ticket #20 
Ticket #21 
Ticket #22 

1 View comments list
2. Add Comment
3. Edit Comment
4. Delete Comment
5. Details page for comment
6. only owner of comment can edit and delete the comment

## Type of change

# Testing Instructions
1. fetch branch sf-initial
2. run Tabloid app
3. npm start in terminal

4. View comments list for a specific post 
 - click view comment button on post details page
 - view all the comments for that post in order of date (most recent on top)
- only owner of comment should be able to see edit and delete buttons (sign in as another user to test)

5. Add comments for that specific post 
  - click add comment button on post details page
 -  add subject and content
 - click save and be brought back to comments list for post
 - click cancel and be brought back to comments list for post

6. Edit comment 
 - on comments list page, on comment card, click edit button
 - edit comment and click save (brought to comments details page)
 - if click cancel, brought back to comments list


7. Delete comment 
 - on comments list page, cn comment card, click delete button
 - confirm delete and be brought back to comments list
- cancel delete and be brought back to comments list


# Checklist:
